examples/inputs/emgauss.yaml

```diff
@@ -47,7 +47,8 @@ wake:
     velocity_model: empirical_gauss
 
   enable_secondary_steering: false
-  enable_yaw_added_recovery: true
+  enable_yaw_added_recovery: false
```

Comment:
Should `enable_yaw_added_recovery` be changed here to `false`? Ideally, we'd be able to leave this as `true`